File: src/main/java/models/LoginPage.java
package models;

public class LoginPage
{
    private String username;
    private String password;

    public LoginPage(String username, String password)
    {
        this.username = username;
        this.password = password;
    }

    public LoginPage() {

    }

    public void login_details(String username)
    {
        this.username = username;
        this.password = password;
    }
}

File: models/login_page.py
class LoginPage:

    def __init__(self, username, password):
        self.username = username
        self.password = password

    def __repr__(self):
        return str(
            {
                "Username" : self.username,
                "Password" : self.password
            }
        )

    def json(self):
        return {
            "Username" : self.username,
            "Password" : self.password
        }